### PR TITLE
Cleaned up code

### PR DESCRIPTION
**Pull request by sethharling**
cleaned the following files:
Update.java
ReadAllOrders.java
ReadOneOrder.java
Main.java (under model)

### CODE REVIEW

**Review by snowjtSD — COMMENTED** [review]
Hey, looks good man, I'm sure Jessy may look at this, but the wrapping standard is at 100 char I believe per line and that's the only change I would recommend, the rest is great

File: app/src/main/java/com/yagni/model/ReadAllOrders.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import java.sql.*;
 import javax.swing.*;
 import javax.swing.table.DefaultTableModel;
+
```
Comment (S4kRiPp4) [inline app/src/main/java/com/yagni/model/ReadAllOrders.java]: delete line 6 whitespace. import com.yagni.controller.*; should be grouped with other imports

File: app/src/main/java/com/yagni/model/ReadAllOrders.java [hunk 2]
```diff
@@ -16,7 +17,6 @@ public ReadAllOrders(JTable ordersTableIn){
     public void readAllOrders(DbConnection linkDB) {
         
         try {
-            
             Statement state = linkDB.getConnection().createStatement(); // creates a statment object
```
Comment (S4kRiPp4) [inline app/src/main/java/com/yagni/model/ReadAllOrders.java]: move your comment that trails the code at line 20 up, the comment should be on line 20 and the code on line 21. Also, statement is misspelled in the comment.
like this:
line 20 --> // creates a statement object
line 21 -->Statement state = linkDB.getConnection().createStatement();

File: app/src/main/java/com/yagni/model/ReadOneOrder.java [hunk 3]
```diff
@@ -26,13 +27,12 @@ public void readOneOrder(DbConnection linkDb){
             // creates a DefaultTableModel object
             DefaultTableModel tableModel = (DefaultTableModel) ordersTable.getModel();
```
Comment (S4kRiPp4) [inline app/src/main/java/com/yagni/model/ReadOneOrder.java:29]: delete whitespace @ line 28. moving the current line 29, up to line 28

File: app/src/main/java/com/yagni/model/ReadOneOrder.java [hunk 4]
```diff
@@ -47,10 +47,10 @@ public void readOneOrder(DbConnection linkDb){
             productQuantity = result.getString(6);
             String [] row = {orderNum, orderDate, hashEmail, location, productID, productQuantity};
             tableModel.addRow(row);
-            
+
```
Comment (S4kRiPp4) [inline app/src/main/java/com/yagni/model/ReadOneOrder.java:50]: delete the whitespace between your catch exception and your closing curly on the while statement.

**Review by S4kRiPp4 — APPROVED** [review]
I made all the changes that I seen w/ my own commits to get these PR's all merged today.
I also pull this PR multiple times locally to confirm the changes and ran a spell check.

I approve and next person to review can go ahead and merge unless they see something critical that needs a change.

**Review by KyleZim97 — APPROVED** [review]
Clean up looks good.

Spoke with Jesse about some of this code and we got it straitened out.

Looks Good.

**Pull request merged**